src/asciidoctor-extensions-api.js

```diff
@@ -45,14 +48,19 @@ var toBlock = function (block) {
  */
 var Extensions = Opal.Asciidoctor.$$scope.Extensions;
 
+// Alias
+Opal.Asciidoctor.Extensions = Extensions;
+
 /**
  * @memberof Extensions
  */
 Extensions.create = function (name, block) {
   if (typeof name === 'function' && typeof block === 'undefined') {
-    return Opal.send(this, 'build_registry', null, toBlock(name));
+    return Opal.send(this, 'create', null, toBlock(name));
```

Comment:
The reason I mapped to build_registry was so that we can still build against 1.5.5. Are we still doing that?